ansible/roles/vm_set/files/mux_simulator.py

```diff
@@ -22,6 +22,11 @@
 app = Flask(__name__)
 
 
+TOR_A = 'upper_tor'
```

Comment:
Could we also change these variable names?
```suggestion
UPPER_TOR = 'upper_tor'
```